code/workspaces/infrastructure-api/resources/jupyter.headless-service.template.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
This is actually a headless service for spark rather than jupyter so the file name should probably reflect that. `spark-driver-headless-service`